src/modules/interactive.js

```diff
@@ -0,0 +1,23 @@
+const clearCompletedItems = (container, myTasks, updateIndexes, storeItems) => {
+  const completedItems = container.querySelectorAll('.completed');
+
+  completedItems.forEach((completedItem) => {
+    const listItem = completedItem.closest('.list');
+    const index = Array.from(container.children).indexOf(listItem);
+
+    listItem.remove();
+    myTasks.splice(index, 1);
+  });
+
+  updateIndexes();
+  storeItems();
+};
+// eslint-disable-next-line no-unused-vars
```

Comment:
- Please remove all disabled linter checks, remember linters help us ensure we write code that is clean and easy to read and maintain. Please fix this 👍 👍